pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -156,16 +164,21 @@ func Test_EnforceDeviceUmountPolicy_Removes_Device_Entries(t *testing.T) {
 
 		err := policy.EnforceDeviceMountPolicy(target, rootHash)
 		if err != nil {
+			t.Error(err)
+			t.Errorf("policy state: %+v\n", policy)
```

Comment:
If you add a `t.Logf("policy state: %+v\n", policy)` to the beginning, then it will only be showed if `-v` is used or the test fails, so you dont have to reprint it everywhere on failure